HW6/6.2.py

```diff
@@ -0,0 +1,39 @@
+# Author: Serhii
+# HW5 task 5.2
+
+# Init variables
+login_realist = input("Hey, Please enter your login: ")
+login_desired = 'First'
+# Writing your Login into secure hidden file
+file = ".login.txt"
+print ("Writing your first realistic Login into secure and hidden Linux file: ", file)
+with open('.login.txt', 'w') as f:
+    f.write(login_realist)
+    f.close()
+
+print("===========  Way-1  ===========")
+while not login_realist == "First":
```

Comment:
good job
another way 
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
    
else:
    print('Greeting. Access granted!!!', user_name)